bioblend/galaxy/workflows/__init__.py

```diff
@@ -632,13 +632,25 @@ def show_invocation(self, workflow_id, invocation_id):
         url = self._invocation_url(workflow_id, invocation_id)
         return self._get(url=url)
 
-    def get_invocations(self, workflow_id):
+    def get_invocations(self, workflow_id, history_id=None, user_id=None, include_terminal=True, limit=None, view='collection', step_details=False):
         """
         Get a list containing all the workflow invocations corresponding to the
         specified workflow.
 
         :type workflow_id: str
         :param workflow_id: Encoded workflow ID
+        :type history_id: str
+        :param history_id: Encoded history ID
+        :type user_id: str
+        :param user_id: TODO
```

Comment:
Are you planning to fill up the TODOs, so still WIP?

Reply:
I'll need a dev or someone more familiar with the api to do this.